src/Controls/src/Build.Tasks/CreateObjectVisitor.cs

```diff
@@ -363,12 +363,11 @@ static bool IsXaml2009LanguagePrimitive(IElementNode node)
 		{
 			if (node.NamespaceURI == XamlParser.X2009Uri)
 			{
-				var n = node.XmlType.Name.Split(':')[1];
-				return n != "Array";
+				return node.XmlType.Name != "Array";
```

Comment:
Was the splitting on `:` here important?

We could maybe undo this change and put in a separate PR, if it's not related to the source generator.